### ISSUE TITLE
[Postgres Dialect] `PRIMARY KEY` keyword treated as identifiers

### ISSUE THREAD

**Issue by ackintosh** [sqlfluff/sqlfluff#718]
## Expected Behaviour

```sql
CREATE TABLE main.sample(
    id char(4) NOT NULL,
    label text NOT NULL,
    title text NOT NULL,
    PRIMARY KEY(id)
);
```

Linting the DDL statement would be success.

## Observed Behaviour

Unfortunately linting is failed. I would guess that `PRIMARY KEY` was not treated as keyword. 🤔 

```shell
sqlfluff lint --dialect postgres sample.sql
== [sample.sql] FAIL
L:   5 | P:   5 | L014 | Inconsistent capitalisation of unquoted identifiers.
L:   5 | P:   5 | L029 | Keywords should not be used as identifiers.
```

If we change the DDL statement as below, linting will success.

```sql
CREATE TABLE main.sample(
    id char(4) NOT NULL PRIMARY KEY,
    label text NOT NULL PRIMARY KEY,
    title text NOT NULL
);
```

## Steps to Reproduce

Run lint with the DDL statement I've posted first on this.

## Version
Include the output of `sqlfluff --version` along with your Python version

```shell
$ sqlfluff --version
sqlfluff, version 0.3.6

$ python -V
Python 3.9.1
```

## Configuration

No configurations, using default.

**Comment by ackintosh** [comment]
@sti0 Thanks for the fix! I'm looking forward to it to be released. ✨
